E/eccodes/build_tarballs.jl

```diff
@@ -3,11 +3,11 @@
 using BinaryBuilder, Pkg
 
 name = "eccodes"
-version = v"2.28.0"
+version = v"2.36.0"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://confluence.ecmwf.int/download/attachments/45757960/eccodes-$version-Source.tar.gz", "2831347b1517af9ebd70dd3cad88ae818a8448d4e6c8671aa728617e73431cd5"),
+    ArchiveSource("https://confluence.ecmwf.int/download/attachments/45757960/eccodes-2.36.0-Source.tar.gz", "da74143a64b2beea25ea27c63875bc8ec294e69e5bd0887802040eb04151d79a"),
```

Comment:
Why $version was removed?

Reply:
Ah, forgot to fix up after copy/pasting the URL, thanks!